natives.json

```diff
@@ -34224,15 +34224,15 @@
 			"build": "1207"
 		},
 		"0xCBB7B6EDFA933ADE": {
-			"name": "_0xCBB7B6EDFA933ADE",
+			"name": "_ITEMDATABASE_DESTROY_ITEM_COLLECTION",
 			"comment": "",
 			"params": [
 				{
-					"type": "Any",
-					"name": "p0"
+					"type": "int",
+					"name": "collection"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "void",
```

Comment:
This actually returns ``BOOL``, please don't ever change a non-``void`` type to ``void``.